### PR TITLE
Hide mini tablet immediately upon near grabbing an entity (RC74)

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
Addresses the near-grab aspect of the issue reported here - https://highfidelity.manuscript.com/f/cases/19308/

### CODE REVIEW

**Comment by ctrlaltdavid** [comment]
### Test Plan

Create an entity that you can near grab.

1) Check full-size mini tablet disappears immediately when you near grab an entity.
- Place a hand near the entity.
- Orient your hand and your view so that the mini tablet displays.
- Wait for the mini tablet to fully expand to mini tablet size.
- Squeeze the trigger to grab the entity.
- The mini tablet should immediately disappear (rather than shrinking).

2) Check showing mini tablet disappears immediately when you near grab an entity.
- Per 1) except squeeze the trigger to grab the entity when the mini tablet is expanding from nothing to mini tablet size.
- The mini tablet should immediately disappear (rather than continuing to expand to mini tablet size then disappearing).

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13869/HighFidelity-Beta-PR14208-057ec42.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13869/HighFidelity-Beta-PR14208-057ec42.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13869/HighFidelity-Beta-Interface-PR14208-057ec42.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13869/HighFidelity-Beta-PR14208-057ec42.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13869/HighFidelity-Beta-Interface-PR14208-057ec42.exe

**Comment by qliejahner** [comment]
Currently testing

**Comment by conklin94122** [comment]
Project Approved, got clarification on the different aspects of the originally reported issue.

File: scripts/system/miniTablet.js [hunk 1]
```diff
@@ -973,6 +991,21 @@
...
+        switch (message.action) {
+            case "grab":
+                grabbingHand = HAND_NAMES.indexOf(message.joint);
+                grabbedItem = message.grabbedEntity;
+                break;
+            case "release":
+                grabbingHand = NO_HAND;
+                grabbedItem = null;
+                break;
+            default:
```
Comment (sethalves) [inline scripts/system/miniTablet.js]: do we care about "equip" here?

File: scripts/system/miniTablet.js [hunk 2]
```diff
@@ -973,6 +991,21 @@
...
+                error("Unexpected grab message!");
```
Comment (sethalves) [inline scripts/system/miniTablet.js]: what does error() do?  should this be print?  or do nothing?

**Pull request merged**